crd-docs/crd/kiali.io_kialis.yaml

```
@@ -1248,6 +1248,14 @@ spec:
                                 expression:
                                   type: string
                                   description: "The hide expression."
+                          settings:
+                            description: "Various presentation options."
+                            type: object
+                            properties:
+                              animation:
+                                description: "The traffic animation style. Value must be one of: `dash` or `point`. Default is `point`."
+                                type: string
+                                enum: ["dash", "point" ]
```

Egad! No leading space in the front, but leading space at the end? UG-LY.

```suggestion
                                enum: ["dash", "point"]
```